crates/humanode-runtime/src/frontier_precompiles.rs

```diff
@@ -76,6 +77,8 @@ pub mod precompiles_constants {
     pub const NATIVE_CURRENCY: u64 = 2050;
     /// `CurrencySwap` precompile constant.
     pub const CURRENCY_SWAP: u64 = 2304;
```

Comment:
We'll have to eventually deprecate this one; how would we do it?

Reply:
Yep, the interfaces are compatible. So, just remove the old one and leave the same precompile code ?

Reply:
Done